projects/reliable-integration/0001-high-level-design.md

```diff
@@ -87,185 +89,188 @@ All events will have the following structure:
+## Event Types
 
-### NEW_TX_ACCEPTED
+Events described here are a subset of all events in the `HathorEvents` enum. The event manager only subscribes and handles the ones listed below.
 
-It will be triggered when the transaction is synced, and the consensus algorithm immediately identifies it as an accepted TX that can be placed in the mempool. `tx` datatype is going to be sent. We will reuse the `NETWORK_NEW_TX_ACCEPTED` Hathor Event that is already triggered.
+- `LOAD_STARTED`
+- `LOAD_FINISHED`
+- `NEW_VERTEX_ACCEPTED`
+- `REORG_STARTED`
+- `REORG_FINISHED`
+- `VERTEX_METADATA_CHANGED`
 
-### NEW_TX_VOIDED
+### LOAD_STARTED
 
-It will be triggered inside [this method](https://github.com/HathorNetwork/hathor-core/blob/d39fd0d9515c72b5d68b36594294dbd7a155e365/hathor/consensus.py#L967) when the transaction is received and synced, but the consensus immediately identifies it as a voided TX (As long as a reorg is not in progress). `tx` datatype is going to be sent.
+It will be triggered when the full-node is initializing and is reading locally from the dabatase, at the same time of `MANAGER_ON_START` Hathor event. It should have an empty body.
 
-### NEW_BEST_BLOCK_FOUND
+### LOAD_FINISHED
 
-It will be triggered when a block is found, received by the full node and immediately identified as a valid block by the consensus algorithm. We will reuse the `NETWORK_NEW_TX_ACCEPTED` Hathor event that is already triggered. In addition, it will trigger ```TX_METADATA_CHANGED``` events for all transactions being directly confirmed by this block (Changing the ```first_block``` information). `tx` datatype is going to  be sent.
+It will be triggered when the full-node is ready to establish new connections, sync, and exchange transactions, at the same that when the manager state changes to `READY` [here](https://github.com/HathorNetwork/hathor-core/blob/85206cb631b609a5680e276e4db8cffbb418eb88/hathor/manager.py#L652). Other events will be triggered ONLY after this one, if the `--emit-load-events` flag is not enabled. `EmptyData` is sent.
 
-### NEW_ORPHAN_BLOCK_FOUND
+### NEW_VERTEX_ACCEPTED
 
-It will be triggered inside [this method](https://github.com/HathorNetwork/hathor-core/blob/d39fd0d9515c72b5d68b36594294dbd7a155e365/hathor/consensus.py#L443) when a block is found, received by the full node, but immediately identified as a block that does not belong to the best chain. `tx` datatype is going to be sent.
+It will be triggered when the transaction is synced, and the consensus algorithm immediately identifies it as an accepted TX that can be placed in the mempool. `TxData` is going to be sent. We will reuse the `NETWORK_NEW_TX_ACCEPTED` Hathor Event that is already triggered.
```

Comment:
Maybe clarify that this event will NOT be emitted for partially validated transactions.

Reply:
Thanks!